python/tvm/topi/hexagon/utils.py

```diff
@@ -150,4 +156,91 @@ def get_layout_transform_fn(layout):
         return nc_2048_2d
     if layout == "nhwc-8h8w32c-2d":
         return nhwc_8h8w32c_2d
+    if layout == "n11c-2048c-2d":
+        return n11c_2048c_2d
     raise RuntimeError(f"Unexpected layout '{layout}'")
+
+
+def get_fixed_point_value(flp, dtype="int16"):
```

Comment:
I don't notice anything in the function body (or docs) indicating if/how this function handles `flp` values that are:
- denormalized
- positive or negative infinity
- NaN

Should this function handle those cases gracefully or at least assert if they're encountered?

I think regardless of how they're handled, the docstring should discuss the issue.

Reply:
Good point! Thanks! The function handles denormalized values, but doesn't handle Nan or infinity. I'll add assert for these two cases. Will also include additional details on the denormalized case.